syntax = "proto3";

option java_multiple_files = true;
option java_package = "com.gtri.scrimmage";

package scrimmage_msgs;

message SeedMessage {        
    uint64 sim_seed = 1;
}
